src/plugins/chart_expressions/expression_xy/public/components/tooltip/tooltip.scss

```diff
@@ -22,9 +22,14 @@
   }
 }
 
+.detailedTooltip__labelContainer {
+  max-width: $euiSizeXL * 5;
+}
+
 .detailedTooltip__labelContainer,
 .detailedTooltip__valueContainer {
   overflow-wrap: break-word;
+  word-break: break-all;
```

Comment:
With the suggestion in my previous comment, I believe this `word-break` style can be removed, as the existing `overflow-wrap` style should now work properly.